static/js/tools/scatter/app.test.tsx

```diff
@@ -69,8 +85,18 @@ function mockAxios(): void {
       },
     });
 
-  // Population and statvar data
+  // Statvar data
+  when(axios.get)
```

Comment:
curious why this mocked call is necessary because I believe `/api/variable-group/info` should always be called with the dcid of the variable group we are trying to get info for

Reply:
Good catch- verified that the test passes without this.